attestation-ca/src/lib.rs

```diff
@@ -24,11 +40,11 @@ pub struct SerialisableAttestationCa {
 )]
 pub struct AttestationCa {
     /// The x509 root CA of the attestation chain that a security key will be attested to.
-    pub ca: x509::X509,
+    ca: x509::X509,
     /// If not empty, the set of acceptable AAGUIDS (Device Ids) that are allowed to be
     /// attested as trusted by this CA. AAGUIDS that are not in this set, but signed by
     /// this CA will NOT be trusted.
-    pub aaguids: BTreeSet<Uuid>,
+    aaguids: BTreeMap<Uuid, DeviceDescription>,
```

Comment:
Update `webauthn-authenticator-rs/src/softtoken.rs:930:13` and `webauthn-authenticator-rs/src/softtoken.rs:1012:13` for this